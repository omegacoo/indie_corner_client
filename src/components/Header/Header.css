.Header {
    text-align: center;
    border: 1px solid black;
    height: 10vh;
}

.Header-title {
    display: block;
    width: fit-content;
    margin-left: auto;
    margin-right: auto;
    margin-top: 2vh;
    text-decoration: none;
    color: black;
    padding: 2vh;
}