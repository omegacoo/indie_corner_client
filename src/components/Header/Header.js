import React from 'react';
import { Link } from 'react-router-dom';

import './Header.css';

export default class Header extends React.Component {

    render(){
        return(
            <div className='Header'>
                <Link 
                    className='Header-title'
                    to={'/'}
                >
                    Indie Corner
                </Link>
            </div>
        );
    };
};