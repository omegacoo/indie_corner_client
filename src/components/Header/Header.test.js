import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Header from './Header';

describe('Header component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(
            <BrowserRouter>
                <Header />
            </BrowserRouter>, 
            div
        );
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders UI correctly', () => {
        expect(
            renderer.create(
                <BrowserRouter>
                    <Header />
                </BrowserRouter>).toJSON()
        ).toMatchSnapshot();
    });
});