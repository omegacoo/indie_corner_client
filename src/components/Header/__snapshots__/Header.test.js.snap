// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders UI correctly 1`] = `
<div
  className="Header"
>
  <a
    className="Header-title"
    href="/"
    onClick={[Function]}
  >
    Indie Corner
  </a>
</div>
`;
